config/services/rebalancer/service.json

```diff
@@ -1,6 +1,7 @@
 {
 	"params": {
 		"networks": [ "manta", "admin" ],
-		"ram": 512
+		"ram": 1024,
+		"delegated_dataset": true
```

Comment:
should be "delegate_dataset" (there was an extra 'd')

Reply:
I `s/delegated/delegate/`.  But oddly it works with `delegated_dataset`

Reply:
Ah actually all manta zones already have a delegate dataset because they inherit the [manta application params](https://github.com/joyent/sdc-manta/blob/master/config/application.json#L4). That's why it is there even with the misspelled property because it's already there under the zone root /data with or without this PR. We can perhaps skip specifying it in the service.json.

Reply:
Ah, that makes sense.  Updated the PR.